app/components/agenda/agendaitem/agendaitem-decision-edit.js

```diff
@@ -4,27 +4,36 @@ import { tracked } from '@glimmer/tracking';
 import { task } from 'ember-concurrency';
 
 export default class AgendaitemDecisionEditComponent extends Component {
-  @tracked decisionActivity = this.args.decisionActivity;
+  @tracked decisionResultCode;
+
+  constructor() {
+    super(...arguments);
+    this.loadDecisionActivityResult.perform();
+  }
+
+  @task
+  *loadDecisionActivityResult() {
+    this.decisionResultCode = yield this.args.decisionActivity?.decisionResultCode;
```

Comment:
`decisionActivity` shouldn't be optional here, since the save would fail if it is. I copied this from somewhere.